src/ui/devtools/components/body.tsx

```diff
@@ -30,11 +30,23 @@ function Body({data, actions}: BodyProps) {
             reset: () => actions.resetDevDynamicThemeFixes(),
         });
 
-    function onTextRender(node) {
+    function onTextRender(node: HTMLTextAreaElement) {
         textNode = node;
         if (!state.errorText) {
             textNode.value = wrapper.fixesText;
         }
+        node.addEventListener('keydown', (e) => {
+            if (e.key === 'Tab') {
+                e.preventDefault();
+                const start = node.selectionStart;
+                const end = node.selectionEnd;
+                const before = node.value.substring(0, start);
+                const after = node.value.substring(end);
+                node.focus();
+                node.value = `${before}  ${after}`;
+                node.setSelectionRange(start + 1, start + 1);
```

Comment:
Fix the cursor position, play with the selection range to place it after the inserted 4 spaces.